local-artifact-mirror/Makefile

```diff
@@ -62,11 +62,25 @@ build-and-push-local-artifact-mirror-image:
 .PHONY: build-and-push-local-artifact-mirror-image-chainguard
 build-and-push-local-artifact-mirror-image-chainguard:
 	mkdir -p build
-	dagger call publish-local-artifact-mirror-image \
-		--ec-version $(PACKAGE_VERSION) \
-		--repo $(IMAGE_NAME) \
-		--arch $(ARCH) \
-		> build/digest
+ifdef REGISTRY_SERVER
+	dagger call \
+		with-registry-login \
```

Comment:
nit: why not just have 2 dagger calls, and only call the registry one if `REGISTRY_SERVER` is set.